source/common/common/stl_helpers.h

```diff
@@ -13,3 +17,14 @@ template <class Container, class T> bool containsReference(const Container& c, c
          }) != c.end();
 }
 } // namespace Envoy
+
+// NOLINT(namespace-envoy)
+// Overload functions in std library.
+namespace std {
```

Comment:
Can we put it into the global namespace? I think it can be used everywhere that way.

Reply:
in std namespace can also be used everywhere, at least in Envoy and quic namespaces as tested.
Similar overloads are all in std namespace in envoy. I just follow the convention.